bears/general/LineLengthBear.py

```diff
@@ -30,7 +47,30 @@ def run(self,
         :param indent_size:         Number of spaces per indentation level.
         :param ignore_length_regex: Lines matching each of the regular
                                     expressions in this list will be ignored.
+        :param language:            Programming language of the source code.
         '''
+        if language:
+            try:
+                language = Language[language]
+            except UnknownLanguageError:
+                logging.warn(
+                    language + ' is an invalid language name '
+                    'or is not recognized by coala. Language specific '
+                    'line length limitation can not be applied.'
+                )
+                language = Language['Unknown']
+            else:
+                if 'max_line_length' in language.attributes:
+                    max_line_length = language.max_line_length
+                else:
+                    logging.warn(
+                        'Language ' + str(language) + ' does not have '
+                        '`maximum line length` defined in coalang. '
+                        'Falling back to 80 characters as maximum line length.'
```

Comment:
no.  we fall back to the *setting* , which will be in the arg `max_line_length`, which defaults to 79 cpl.

But most Language will never have a `max_line_length`, because most languages dont restrict that, so this warning is not user friendly, nor is it developer friendly.  We dont want patches for adding `max_line_length` to language definitions unless the language does restrict line length, and we know that is rare.

Reply:
Okay, I'll remove the warning :)